ledger/participant-integration-api/src/main/scala/platform/store/CompletionFromTransaction.scala

```diff
@@ -28,10 +28,10 @@ private[platform] object CompletionFromTransaction {
       commandId: String,
       transactionId: String,
       applicationId: String,
-      maybeSubmissionId: Option[String] = None,
-      maybeDeduplicationOffset: Option[String] = None,
-      maybeDeduplicationDurationSeconds: Option[Long] = None,
-      maybeDeduplicationDurationNanos: Option[Int] = None,
+      optSubmissionId: Option[String] = None,
```

Comment:
I'm just wondering: have we defined a standard naming convention for `Option`s in the Daml SDK?

Reply:
The result of a `: Option[` text search yields mostly no prefix, but `opt` in a few places (I guess where it's important to note), i.e., it doesn't look like we have a convention but we rather rely on ad-hoc choices to try and make the code more readable. Here the prefix is present because it mirrors the corresponding (nullable) DB field types, as `CompletionFromTransaction` is basically a row-to-API-proto conversion utility.